catalogs/HAND/main.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cd $(dirname "$0")
+
+python3 build_catalog.py --root-uri https://hand-data.s3.amazonaws.com
```

Comment:
Should this be an S3 URL?

Reply:
I don't think so, but now I'm not sure. This came from this line, which is removed in this diff:

https://github.com/azavea/noaa-flood-mapping/pull/72/files#diff-1bfef6bb952b73fa75a183fa9f2545b5L18

Reply:
Looks like its just used to set asset href urls. So it could be https or s3. When I asked about this in our stac slack channel I got this response, which makes sense:

> IMO if they are publicly accessible it would be preferable to use the https scheme for easier client access. If you need credentials it would make sense to keep them as s3://

So, probably actually s3? Buuuut, I didn't change here what it was so lets defer a change until later if necessary.